apps/core/lib/core/event_repo.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Core.EventRepo do
+  @type operation ::
+          :user_created
+          | :user_updated
+          | :user_deleted
+          | :post_created
+
+@spec save_event(operation, {pos_integer(), String.t(), any()}) :: :ok | {:error, any()}
+  def save_event(:user_created, {user_id, user_email, time}),
```

Comment:
Is this really `user_email`? The call site (here: https://github.com/smt116/elixir-umbrella-sample-application/pull/4/files#diff-c51367e2ad9ecfed2e03cec7cb6e1b17R28 ) passes a map as the second argument (`Map.delete(attrs, "password")`).